DATA/production/qc-async/trd.json

```diff
@@ -62,6 +62,35 @@
           "pulseheightpeaklower": "1.0",
           "pulseheightpeakupper": "5.0"
         }
+      },
+      "PulseHeightTrackMatch": {
+        "active": "false",
```

Comment:
Why add a deactivated task?

Reply:
simplicity of all the code, so the json files are the same, it makes the checking easier. If its a problem i can remove.

Reply:
Ok, if that is easier for you, then sure, no problem.